docs/source/sysadmin_guide/grafana_configuration.rst

```diff
@@ -0,0 +1,150 @@
+********************************
+Configuring Grafana and InfluxDB
+********************************
+
+This procedure describes how to display Sawtooth network statistics and operating system metrics in Grafana, using Docker, InfluxDB, and Telegraf.
+
+Requirements
+============
+
+This guide assumes Docker and one or more sawtooth validators have been configured.
+
+Download sawtooth-core from the Hyperledger GitHub repository.
+
+.. code-block:: console
+
+   $ git clone https://github.com/hyperledger/sawtooth-core.git
+
+InfluxDB
+========
+
+#. First, pull the InfluxDB Docker container from Docker Hub.
+
+   .. code-block:: console
+
+     $ docker pull influxdb
+
+#. Next, start the InfluxDB Docker container with persistent storage on the local file system and port 8086 exposed to the network. We also configure the database, admin, and regular users here.
+
+   .. note::
+
+      Depending on the host operating system /var/lib/influx-data may not be appropriate, modify this path accordingly.
+
+   .. note::
+
+      Be sure to replace these passwords with something more secure! Some characters such as ``,@!$`` have special meanings in the shell and may cause unexpected results if not properly escaped in the following command.
+
+   .. code-block:: console
+
+     $ sudo mkdir /var/lib/influx-data
+     $ docker run -d -p 8086:8086 -v /var/lib/influx-data:/var/lib/influxdb \
+       -e INFLUXDB_DB=metrics -e INFLUXDB_HTTP_AUTH_ENABLED=true \
+       -e INFLUXDB_ADMIN_USER=admin -e INFLUXDB_ADMIN_PASSWORD='admin' \
+       -e INFLUXDB_USER=lrdata -e INFLUXDB_USER_PASSWORD='test' \
+       --name sawtooth-stats-influxdb influxdb
+
+Grafana
+=======
+
+#. First, build the Grafana Docker image from the Dockerfile included in the sawtooth-core repository.
+
+   .. code-block:: console
+
+     $ cd sawtooth-core/docker
+     $ docker build . -f grafana/sawtooth-stats-grafana -t sawtooth-stats-grafana
+
+#. Run the Grafana container.
+
+   .. code-block:: console
+
+     $ docker run -d -p 3000:3000 --name sawtooth-stats-grafana sawtooth-stats-grafana
+
+#. Next, open the Grafana web page, which should now be accessible by IP address or FQDN at http://{host}:3000.
```

Comment:
Is there a default value you can give here instead of `{host}`? @chenette and I have discussed this kind of syntax before and the consensus seems to be that it is confusing and a default that can be copy-pasted is better.

Reply:
I don't see any good value to default to here, 127.0.0.1 would be the only reasonable address to assume, and that would only be right if you are running these docker containers on the same machine you are reading the guide from.

Reply:
How about showing "localhost:3000", with a note after the command to change it if you're not running the browser on the same host as the Grafana software?

Reply:
I added a note to replace {host} with the IP or FQDN of the system running the container.